books/urls.py

```diff
@@ -23,6 +23,7 @@
     path('robots.txt', support.robots_txt),
     path('sitemap.txt', support.sitemap),
     path('data.json', support.get_data_json),
+    path('_ah/warmup', support.get_data_json),
```

Comment:
Add a comment with a link to the doc page about warmup requests. Otherwise it's unclear what this odd path is fore.

Reply:
Make sense